app/scripts/app.js

```diff
@@ -485,6 +485,80 @@ angular
       LabelFilter.readPersistedState();
     });
   })
+  .run(function($location, $uibModal, AuthService, Constants) {
+    if (Math.sign(Constants.INACTIVITY_TIMEOUT_PERIOD) < 1 || $location.path() === '/logout' || $location.path() === '/error') {
+      return false;
+    }
+    var isLocalLogoutModalShown = false;
+    var updateLastInteraction = function() {
+      localStorage.setItem('origin-web-console-last-interaction-timestamp', new Date());
+    };
+    var updateLogOut = function(bool) {
+      localStorage.setItem('origin-web-console-inactivity-logout', bool);
```

Comment:
you need to make sure this localStorage gets reset on login, the auth service has something like an onUserChanged i think that you could listen to

Reply:
How about clearing the localStorage on logout ? Wouldnt that be enough ?

Reply:
no because there may be other tabs that haven't yet seen the flag to logout, think if you have 3+ tabs open

Reply:
That would also handle the case you mentioned bellow 
> you probably want to make sure that clicking the logout link does not reset the last interaction time, stopping propagation of the event from the ng-click of the button would probably fix this.